Content.Server/Blob/BlobCoreSystem.cs

```diff
@@ -0,0 +1,338 @@
+            return;
+
+        CreateBlobObserver(uid, args.Player.UserId, component);
+    }
+
+    public bool CreateBlobObserver(EntityUid blobCoreUid, NetUserId userId, BlobCoreComponent? core = null)
+    {
+        var xform = Transform(blobCoreUid);
+
+        if (!Resolve(blobCoreUid, ref core))
+            return false;
+
+        var blobRule = EntityQuery<BlobRuleComponent>().FirstOrDefault();
+
+        if (blobRule == null)
+        {
+            _gameTicker.StartGameRule("Blob", out var ruleEntity);
+            blobRule = Comp<BlobRuleComponent>(ruleEntity);
+        }
+
+        var observer = Spawn(core.ObserverBlobPrototype, xform.Coordinates);
+
+        core.Observer = observer;
+
+        if (!TryComp<BlobObserverComponent>(observer, out var blobObserverComponent))
+            return false;
+
+        blobObserverComponent.Core = blobCoreUid;
+
+        _mindSystem.TryGetMind(userId, out var mind);
+        if (mind == null)
+            return false;
+
+        _mindSystem.TransferTo(mind, observer, ghostCheckOverride: false);
+
+        _alerts.ShowAlert(observer, AlertType.BlobHealth, (short) Math.Clamp(Math.Round(core.CoreBlobTotalHealth.Float() / 10f), 0, 20));
+
+        var antagPrototype = _prototypeManager.Index<AntagPrototype>(core.AntagBlobPrototypeId);
+        var blobRole = new BlobRole(mind, antagPrototype);
+
+        _mindSystem.AddRole(mind, blobRole);
+        SendBlobBriefing(mind);
+
+        blobRule.Blobs.Add(blobRole);
+
+        if (_prototypeManager.TryIndex<ObjectivePrototype>("BlobCaptureObjective", out var objective)
+            && objective.CanBeAssigned(mind))
+        {
+            _mindSystem.TryAddObjective(blobRole.Mind, objective);
+        }
+
+        if (_mindSystem.TryGetSession(mind, out var session))
+        {
+            _audioSystem.PlayGlobal(core.GreetSoundNotification, session);
+        }
+
+        _blobObserver.UpdateUi(observer, blobObserverComponent);
+
+        return true;
+    }
+
+    private void SendBlobBriefing(Mind.Mind mind)
+    {
+        if (_mindSystem.TryGetSession(mind, out var session))
+        {
+            _chatManager.DispatchServerMessage(session, Loc.GetString("blob-role-greeting"));
+        }
+    }
+
+    private void OnDamaged(EntityUid uid, BlobCoreComponent component, DamageChangedEvent args)
+    {
+        var maxHealth = component.CoreBlobTotalHealth;
+        var currentHealth = maxHealth - args.Damageable.TotalDamage;
+
+        if (component.Observer != null)
+            _alerts.ShowAlert(component.Observer.Value, AlertType.BlobHealth, (short) Math.Clamp(Math.Round(currentHealth.Float() / 10f), 0, 20));
+    }
+
+    private void OnStartup(EntityUid uid, BlobCoreComponent component, ComponentStartup args)
+    {
+        ChangeBlobPoint(uid, 0, component);
+
+        if (TryComp<BlobTileComponent>(uid, out var blobTileComponent))
+        {
+            blobTileComponent.Core = uid;
```

Comment:
In fact don't hardcode this either. Make a `SetBlobCore()` for setting the core, and a `RefreshBlobTileData` for handling refreshing the current chem and, if the chem was changed, firing necessary events to tell other systems like factories, nodes, etc that they should update.